backend/src/core/routes/tests/campaign.routes.test.ts

```diff
@@ -74,6 +75,168 @@ describe('GET /campaigns', () => {
       ]),
     })
   })
+
+  test('List campaigns order by created at', async () => {
```

Comment:
just for learning, I've seen `it` used instead of `test` and was wondering the difference and found this: https://stackoverflow.com/questions/45778192/what-is-the-difference-between-it-and-test-in-jest

TLDR: functionally identical, but use the one that's best for readability